packages/esm-service-queues-app/src/queue-table/queue-table-by-status.component.tsx

```diff
@@ -0,0 +1,103 @@
+import { InlineNotification, Tab, TabList, TabPanel, TabPanels, Tabs } from '@carbon/react';
+import { isDesktop, useLayoutType } from '@openmrs/esm-framework';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from '../active-visits/active-visits-table.scss';
+import { useQueueEntries } from '../hooks/useQueueEntries';
+import { type Concept, type Queue, type QueueTableColumn, type QueueTableTabConfig } from '../types';
+import QueueTableNameCell from './cells/queue-table-name-cell.component';
+import { QueueTable } from './queue-table.component';
+import QueueTableStatusCell from './cells/queue-table-status-cell.component';
+import QueueTablePriorityCell from './cells/queue-table-priority-cell.component';
+import { useNavigate } from 'react-router-dom';
+
+interface QueueTableByStatusProps {
+  queue: Queue;
+  status: Concept;
+  configByStatus?: Map<string, QueueTableTabConfig>;
+  all?: QueueTableTabConfig;
```

Comment:
We probably want to rename `all` to `allStatusTabConfig` or something more explicity.  We should also document how this is intended to be used, what happens by default, etc.